package.json

```diff
@@ -9,6 +9,7 @@
     "url": "https://jku-vds-lab.at/"
   },
   "license": "SEE LICENSE FILE",
+  "main": "dist/index.js",
```

Comment:
This line is necessary, otherwise I cannot import files to in coral_public and cohort